Extras/RR_KerbalismEmitters.cfg

```diff
@@ -0,0 +1,19 @@
+//Add Kerbalism radiation emitters to nuclear engines
+@PART[RREngineNERVA*]:NEEDS[FeatureRadiation]:FOR[KerbalismDefault]
```

Comment:
Never use `:FOR` inside a mod that you don't own. This tells MM to run this as part of the sequence of KerbalismDefault but also tells MM that KerbalismDefault is installed, and will cause patches depending on this in other mods to run, and can cause major conflicts.

MM does not respect NEEDS when FOR is declared. Replace it with BEFORE or AFTER.

Reply:
Gotcha; thought this was the case but then heard conflicting info so just went with first thing that worked.